src/ucp/wireup/wireup.c

```diff
@@ -699,13 +720,15 @@ ucp_wireup_connect_lane(ucp_ep_h ep, const ucp_ep_params_t *params,
             ep->uct_eps[lane] = uct_ep;
         } else {
             uct_ep = ep->uct_eps[lane];
+            ucs_assert(ucp_wireup_ep_test(uct_ep));
         }
 
         ucs_trace("ep %p: connect uct_ep[%d]=%p to addr[%d] wireup", ep, lane,
                   uct_ep, addr_index);
+        connect_aux = !cm_ep_init_flags &&
```

Comment:
check (lane != ucp_ep_get_cm_lane(ep)) instead?

Reply:
nope, cm_lane is connected at line 656, this is connection of TL lane with CM wireup